src/hostname/hostnamed.c

```diff
@@ -208,21 +208,19 @@ static int get_hardware_model(char **ret) {
         return get_dmi_data("ID_MODEL_FROM_DATABASE", "ID_MODEL", ret);
 }
 
-static int get_hardware_serial(char **ret) {
+static int get_hardware_firmware_data(const char *dmi_data, char **ret) {
```

Comment:
Not a matter, but I prefer `dmi_data` -> `attribute`, `attr`, or `sysattr`.

Reply:
preferred `sysattr`